include/Dialect/LWE/IR/LWETypes.td

```diff
@@ -37,6 +37,17 @@ def LWECiphertext : LWE_Type<"LWECiphertext", "lwe_ciphertext"> {
   let assemblyFormat = "`<` struct(params) `>`";
 }
 
+// TODO: Generalize RLWECiphertext
```

Comment:
No need for the TODO here or below on RLWEPlaintext. If you have some specific action you want to leave a note about, please attach it to a github issue (new or existing) such as

```
// TODO(https://github.com/google/heir/issues/337): add support for foo
```

Reply:
I will mark the PR as `pull_ready` after this is resolved.

Reply:
Aha, Thanks!